pkg/operators/oci-handler/oci.go

```diff
@@ -120,6 +121,12 @@ func (o *ociHandler) InstanceParams() api.Params {
 			Description: "List of allowed digests, if image digest is not part of it, execution will be denied. By default, all digests are allowed",
 			TypeHint:    api.TypeString,
 		},
+		{
+			Key:         registry,
```

Comment:
I don't know if this is specific enough. Perhaps `allowed-registries` is better.